extension/data/modules/oldreddit.js

```diff
@@ -24,6 +24,12 @@ function oldReddit () {
         }
     }
 
+    /**
+     * Handles `thing` items as they become visible in the viewport.
+     * @function
+     * @param {IntersectionObserverEntry} entries
```

Comment:
Doc nit: this is an array
```suggestion
     * @param {IntersectionObserverEntry[]} entries
```